packages/flame/lib/src/text/common/line_metrics.dart

```diff
@@ -50,6 +54,9 @@ class LineMetrics {
   double get bottom => baseline + descent;
   double get height => ascent + descent;
 
+  late Vector2 _size = Vector2.zero();
+  Vector2 get size => _size;
```

Comment:
This getter doesn't give anything, just a final `size` variable will have the same effect since `Vector2` is mutable.

Reply:
good point!